operator/metrics/metrics.go

```diff
@@ -190,6 +191,7 @@ func registerMetrics() []prometheus.Collector {
 		Namespace: Namespace,
 		Name:      "ces_queueing_delay_seconds",
 		Help:      "CiliumEndpointSlice queueing delay in seconds",
+		Buckets:   prometheus.ExponentialBuckets(float64(3600)/(1<<19), 2, 20),
```

Comment:
Using hand-picked values looks even nicer:
```suggestion
		Buckets:   append(prometheus.DefBuckets, []float64{30, 60, 120, 240, 600, 1200, 3600}...),
```

Reply:
Agreed, it would be nice to use hand picked buckets, as the numbers are a bit easier for someone to convert IMO. I'm a bit concerned with the number of buckets however, 20 is quite a bit. What's the average and p99 in the larger environments you are running Cilium in?

Reply:
I changed it to be extended by 5 buckets only. We've seen values going into 10s of minutes and expect that it can go even higher.